src/components/drawer/drawer.test.tsx

```diff
@@ -0,0 +1,480 @@
+import React from "react";
+import { act, render, screen, waitFor } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+import Drawer from ".";
+
+describe("Drawer", () => {
```

Comment:
comment (non-blocking): this describe isn't really adding much so we could flatten this an use `test` but it's not a huge deal